src/main/java/org/broad/igv/sam/AlignmentRenderer.java

```diff
@@ -1057,6 +1082,180 @@ private void drawLargeIndelLabel(Graphics2D g, boolean isInsertion, String label
         }
     }
 
+    private void drawInsertions(Rectangle rect, Alignment alignment, RenderContext context, AlignmentTrack.RenderOptions renderOptions,
```

Comment:
@jrobinso this method, drawInsertions, if I understand correctly, was authored outside the scope of this pull request (see Git Blame information). 

I did notice that it, along with the following methods in this class (VariantRenderer) is not used. You may consider removing them:
- drawInsertions
- renderExpandedInsertion
- getOutlierStatus
- compareToBounds

Running "unused declarations" code analysis produces many more unused methods - so I think that this is a bigger issue and might not be related to this pull request.

Reply:
Thanks, could you remove this method?    That will make for a cleaner merge.  Also could you remove the doc as that has been moved to the igv doc repository?  With this I think this will be ready to merge.   Thanks for your prompt responses.